tests/castle-vault.ts

```diff
@@ -1054,7 +1057,7 @@ describe("castle-vault", () => {
     }
 
     function testPortRewardClaiming(subReward: boolean) {
-        const depositQty = 1024502;
+        const depositQty = 3025004087;
```

Comment:
is this number's non-roundedness intentional? when i first started working on the tests these seemed like magic numbers (i.e. i should not change) but they really weren't and just made doing mental math on expected values harder

Reply:
this number is actually just arbitrary.